iterative/kubernetes/provider.go

```diff
@@ -408,6 +413,23 @@ func getInstanceType(instanceType string, instanceGPU string) (map[string]map[st
 		}, nil
 	}
 
+	
+	if match := regexp.MustCompile(`^(\d+)-(\d+)(?:\+([^*]+)\*([1-9]\d*))?$`).FindStringSubmatch(size); match != nil {
```

Comment:
this was pulled from the task code right? can we easily abstract it out so doesn't exist in two places?

Reply:
> _this was pulled from the task code right?_

Yes[^1]

> _can we easily abstract it out so doesn't exist in two places?_

No[^2]

[^1]: With a few minor tweaks.
[^2]: Unless you want to review a dozen pull requests.

Reply:
or are the tweaks different enough that it wouldn't matter?

Reply:
Apart from the regular expression, there is nothing in common. I'd merge this as-is. 😅